src/lib/free-queue/test.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>FreeQueueSAB Tests</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/10.6.0/mocha.min.css" integrity="sha384-BBZj3VfQ5pAok2bd0uXf+8cRAJ3D/JZWrxBlnWx6BcsBO96ZzZR40fpC20MtWbM8" crossorigin="anonymous">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mocha/10.6.0/mocha.min.js" integrity="sha384-A5EqeN0zznOH0u6V4OduoPlHhkhk+Km4eBBNBq0UAE1Z51cFJfX/f9biOo2SmCCS" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/chai/5.1.1/chai.min.js" integrity="sha384-Zg6S9cOeB+JIKb63Z7Qi7N1vGb1nctA+lPjPBmAxFQf3mxxu1BO+e+b62MkzJeEt" crossorigin="anonymous"></script>
```

Comment:
chai.min.js doesn't seem to be available on cdnjs for me.  I just see the full chai.js:
https://cdnjs.com/libraries/chai

Trying to use chai.js I get the error `SyntaxError: Unexpected token 'export'` in chai.js when loading the page.

Reply:
Yes, I resolved it. After the meeting, I tried to re-run the test cases and that error popped up.

Reply:
The tests are running now for me, but I still see the SyntaxError in the console.  It's in chai.js so maybe there's not much we can do?